docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml

```diff
@@ -133,23 +133,45 @@ comment: |
   This question is used to introduce your interview. Please customize
 id: Motion to Reconsider
 question: |
+  Motion to Reconsider, Motion to Modify, or Motion to Reconsider and Modify
+subquestion: |
+  File one of these motions if you need the Appeals Court to look at your case again because:  
+  
+  * the court's reasoning that led to the current order was wrong, 
+  * the court failed to consider evidence, or
+  * the judge should know about new information that affects the decision the court already made. 
```

Comment:
So the way it has been described to us is that a Motion to Reconsider/Modify cannot be used to change the actual decision so I am uncertain we should add this language.